src/chapters/exceptions/exceptions-control-flow.rst

```diff
@@ -0,0 +1,72 @@
+Exceptions-Control-Flow
+=======================
+
+In some instances we receive input from a user, and we can't always ensure they will type in something that will run perfectly with our code.
```

Comment:
"In some instances, we receive input from a user and we cannot always ensure they will type in something that will run with our code."

Reply:
fixed